packages/koenig-lexical/test/e2e/cards/signup-card.test.js

```diff
@@ -25,4 +29,126 @@ test.describe('Signup card', async () => {
             <p><br /></p>
         `, {ignoreCardContents: true});
     });
+
+    test('can edit header', async function ({page}) {
+        await createSignupCard({page});
+
+        await page.keyboard.type('Hello world');
+        const firstEditor = page.locator('[data-kg-card="signup"] [data-kg="editor"]').nth(0);
+        await expect(firstEditor).toHaveText('Hello world');
+    });
+
+    test('can edit subheader', async function ({page}) {
+        await createSignupCard({page});
+
+        await page.keyboard.type('Hello world');
+
+        await page.keyboard.press('Enter');
+        await page.keyboard.type('Hello subheader');
+
+        const firstEditor = page.locator('[data-kg-card="signup"] [data-kg="editor"]').nth(0);
+        const secondEditor = page.locator('[data-kg-card="signup"] [data-kg="editor"]').nth(1);
+
+        await expect(firstEditor).toHaveText('Hello world');
```

Comment:
The header field has been also tested with the previous test – could focus on testing the subheader only here?